style/ruby/.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+AllCops:
+  Exclude:
+    - 'Gemfile'
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'spec/spec_helper.rb'
+    - 'vendor/**/*'
+
+Metrics/LineLength:
+  Max: 120
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
```

Comment:
I wish I had known about https://raw.githubusercontent.com/thoughtbot/hound/master/config/style_guides/ruby.yml earlier. Their list seems a little bit more friendly than the rubocop defaults without being quite as irritating.

That said, I worked with the rubocop list and was generally happy with the result.

It's hard for me to say if the Hound list would expose as many code smells.